### ISSUE TITLE
Build with sandboxed nix

### ISSUE THREAD

**Issue by tkerber** [serokell/ariadne#186]
Currently, the `nix build` fails if sandboxing is enabled, due to the non-fixed-output `stack2nix` call. It would be nice if I could install this on my system without having to disable the sandbox first :)

**Comment by yegortimoshenko** [comment]
@kirelagin is currently working on Nixage, a Stack-like build tool compatible with Nix sandbox: https://github.com/kirelagin/nixage

Ariadne is going to be one of the first projects to be transitioned to the new tool. It will allow us to build Ariadne in a sandbox, and will also drastically reduce build times for small changes.

**Comment by yorickvP** [comment]
In the meantime, we could give you the binary closure or the generated (2mb) .nix file

**Comment by yegortimoshenko** [comment]
That has just landed in #311. However, to make it work, you will need to apply https://github.com/NixOS/nix/pull/2409 to Nix. You can install Nix from our Nixpkgs fork, it has that patch:
```
nix-env -f https://github.com/serokell/nixpkgs/archive/master.tar.gz -iA nix
```
